### PR TITLE
Build fix: replace distribution method with fastlane hockey

### PR DESCRIPTION
**Pull request by Xingyuj**
Build fix: replace distribution method with fastlane hockey

### CODE REVIEW

File: Example_DynamicFramework/Demo/Supporting Files/Demo-Info.plist [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>2</string>
+	<string>1532075099</string>
```
Comment (asyncmind0) [inline Example_DynamicFramework/Demo/Supporting Files/Demo-Info.plist]: whats this ?

File: build.sh [hunk 2]
```diff
@@ -31,16 +29,14 @@ pushd .
...
-fastlane gym --scheme StreetHawkDemo --export_method "ad-hoc" --output_directory "$BUILD_OUTPUTS" --output_name "SHDynamic.ipa" --clean true
+fastlane beta
 
 popd
 
 # ---------------------- upload to hockeyapp ---------------------
 
-/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" -notes_type=markdown -source_path=$PWD -repository_url=https://github.com/StreetHawkSDK/ios -api_token=$HOCKEYAPP_TOKEN -app_id=$HOCKEYAPP_APPID_DYNAMIC build/outputs/SHDynamic.ipa
-/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" -notes_type=markdown -source_path=$PWD -repository_url=https://github.com/StreetHawkSDK/ios -api_token=$HOCKEYAPP_TOKEN -app_id=$HOCKEYAPP_APPID_STATIC build/outputs/SHStatic.ipa
+#/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" -notes_type=markdown -source_path=$PWD -repository_url=https://github.com/StreetHawkSDK/ios -api_token=$HOCKEYAPP_TOKEN -app_id=$HOCKEYAPP_APPID_DYNAMIC build/outputs/SHDynamic.ipa
```
Comment (asyncmind0) [inline build.sh]: why is this commented

**Review by asyncmind0 — CHANGES_REQUESTED** [review]

File: Example_DynamicFramework/Demo/Supporting Files/Demo-Info.plist (hunk 1 above)
Comment (Xingyuj) [reply Example_DynamicFramework/Demo/Supporting Files/Demo-Info.plist]: bundle version

File: Example_DynamicFramework/fastlane/Fastfile [hunk 3]
```diff
@@ -26,15 +26,25 @@ platform :ios do
...
+        export_options: {
+          provisioningProfiles: {
+            "com.streethawk.SHDynamic" => "match AdHoc com.streethawk.SHDynamic"
+          }
+        }
+    	)
+      commit = last_git_commit
+      changelog = changelog_from_git_commits(commits_count: 10, pretty: "• %s\n", merge_commit_filtering: 'exclude_merges')
+      hockey(
+        api_token: "b5adb4c23a934ac3807a8c9958a6daec",
```
Comment (asyncmind0) [inline Example_DynamicFramework/fastlane/Fastfile]: pleas read this frm env variable HOCKEYAPP_TOKEN

File: Example_StaticLibrary/fastlane/Fastfile [hunk 4]
```diff
@@ -26,16 +26,26 @@ platform :ios do
...
+        export_options: {
+          provisioningProfiles: {
+            "com.streethawk.SHStatic" => "match AdHoc com.streethawk.SHStatic"
+          }
+        }
+    	)
+      commit = last_git_commit
+      changelog = changelog_from_git_commits(commits_count: 10, pretty: "• %s\n", merge_commit_filtering: 'exclude_merges')
+      hockey(
+        api_token: "b5adb4c23a934ac3807a8c9958a6daec",
```
Comment (asyncmind0) [inline Example_StaticLibrary/fastlane/Fastfile]: pleas read this frm env variable HOCKEYAPP_TOKEN

**Review by asyncmind0 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**